chocolatey/choco
# (#3097) Remove ancient history

## Description Of Changes

- Remove the warning about upgrading from ancient versions
- Remove all mention of what was added in which 0.x versions

## Motivation and Context

As we no longer support v0.x versions of Chocolatey, it doesn't make a lot of sense for us to keep around warnings or documentation that only pertain to those versions.

## Testing

1. Run the choco installer from this package and verify the warning is not emitted
2. Run `choco help` on a few commands and note the absence of mentions of ancient choco versions
3. Run unit tests
4. Run E2E tests

I don't think we have any tests explicitly verifying the exact content of the help messages (and nor should we, beyond maybe a basic spellcheck).

### Operating Systems Testing

Win11

## Change Types Made



## Change Checklist

* [ ] Documentation has been updated. - not the live doc site till the v2 release probably?

## Related Issue

Fixes #3097
